content/english/python-basics/sets/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Sets"
+draft: false
+weight: 6
+---
+
+Just like a list, a set is a collection of items. The difference between a set and a list is that a set cannot contain duplicates. So, if you have 5 chocolates and you put them in a set, you'll only be left with 1 chocolate as duplicates will be removed. 😢
+
+<img src="../img/sets_example.jpg" width="500"/>
+
+In the image above, the left side represents <b><font color="red">Indoor Games</font></b> (Chess & Table Tennis) while the right side represents the <b><font color="green">Outdoor Games</font></b> (Cricket & Football). The middle represents the common part of both these sets. These are the games that can be played Indoors as well as Outdoors (Basketball & Badminton). The middle part is also known as the intersection between two sets.
+
+{{% notice tip %}}
+A set is created by using the set() function or placing all the elements within a pair of curly braces.
+{{% /notice %}}
+
+```python
+indoorGames = set(["Chess", "Table Tennis", "Basketball", "Badminton"])
+outdoorGames = {"Cricket", "Football", "Basketball", "Badminton"}
+print('** Indoor Games **')
+print(indoorGames)
+print('** Outdoor Games **')
+print(outdoorGames)
+```
+</br>
+
+<b>Output</b>
+
+    ** Indoor Games **
```

Comment:
Please surround the output with ``` instead of indent

```
** Indoor Games **
set(["Chess", "Table Tennis", "Basketball", "Badminton"])
** Outdoor Games **
set(["Cricket", "Football", "Basketball", "Badminton"])
```